tests/e2e/src/pages/base.page.ts

```diff
@@ -47,6 +47,23 @@ export class BasePage {
     await this.world.page?.fill(selector, text, config.increasedTimeout);
   }
 
+  async fillSpecificField(inputIDType: string, inputIDName: string, text: string) {
+    const helper = new Helper(this.world);
+    selector = `//*[@${inputIDType}="${inputIDName}"]`;
+    console.log(await this.world.page?.locator(selector).first().isVisible());
+    if (await this.world.page?.locator(selector).first()) {
+      //if (await this.world.page?.locator(selector).first().isVisible()) {
+      if (text === "clipboard") {
+        const result = await helper.getClipboardValue();
+        await this.fill(selector, result);
+      } else {
+        await this.world.page?.fill(selector, text);
```

Comment:
Please search for any existing methods in your IDE ;) Try to change to fill() from base.page.ts

Reply:
Fixed :)